src/lib/project-loader-hoc.jsx

```diff
@@ -72,7 +72,6 @@ const ProjectLoaderHOC = function (WrappedComponent) {
                 assetHost,
                 projectHost,
                 projectId,
-                reduxProjectId,
```

Comment:
Why is this removed?

Reply:
The line is a bug. It was left there by Ray by accident. Sorry, I should have pointed that out!